biom/tests/test_parse.py

```diff
@@ -283,6 +284,15 @@ def test_parse_adjacency_table_no_header(self):
         obs = Table.from_adjacency(''.join(lines))
         self.assertEqual(obs, exp)
 
+    def test_parse_biom_table_hdf5_accent_utf8_regression(self):
+        tab = example_table.copy()
+        tab.update_ids({i: f'{i}fóó' for i in tab.ids()}, inplace=True)
+        with NamedTemporaryFile(delete=False) as tmpfile:
```

Comment:
Should `delete` be `False`? I think this would leave behind a file on the system.